lib/ui/views/startup_view.dart

```diff
@@ -40,7 +40,7 @@ class StartupView extends HookWidget {
 class StartupViewModel extends ChangeNotifier {
   final log = Logger();
   bool _animationComplete = false;
-  String _destinationRoute = Routes.profilePage;
+  String _destinationRoute = Routes.signin;

```

Comment:
Is this change relevant for the app icon?

Reply:
No, this was for testing on the signup page, I did not commit it last time and accidentally added it here